### PR TITLE
SE-13759: Calling Oracle Stored Procedure with TO_DATE() by DB Connec…

### PR DESCRIPTION
**Pull request by emariotti3**
…tor 1.5.5 aborts 'Timestamp format must be yyyy-mm-dd hh:mm:ss[.fffffffff]'

### CODE REVIEW

**Review by Bardammu — COMMENTED** [review]
Please ask the PM to review this PR

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 1]
```diff
@@ -490,6 +490,29 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
 
+Database connector supports the use of callable statements with the format { call procedureName(:param1, :param2, ..., :paramN) } where parameters will be matched positionally, that is to say, :paramN will match the Nth parameter in the stored procedure declaration. Suppose you have declared a stored procedure which receives two parameters: myFirstParameter and mySecondParameter, in that order. The Database connector does NOT support named parameters in callable statements, so attempting to match parameters by name ( { call myStoredProcedure(mySecondParameter => :param1, myFirstParameter => :param2) } ) will not work.
+
+It also does not support the use of engine specific embedded functions, for example, you might want to change a date to a specific format before calling a procedure in an Oracle database, however, the following approach will not work:  { call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }. All data transformations should be done using dataweave, before the invocation to the callable statement. For example, for the previous example we would perform the following transformation:
+
```
Comment (estebanwasinger) [inline modules/ROOT/pages/db/database-connector-examples.adoc]: This new text should have a header saying something like "Using dates on Stored Procedures".

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 2]
```diff
@@ -490,6 +490,29 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
+				fun format(d: DateTime) = d as String { format: "yyyy-MM-dd HH:mm:ss" }
+				---
+				{ 'date': format(|2019-10-31T13:00:00.000Z|), 'name': 'rick' }
+			]]]>
+			</db:input-parameters>
+		<db:output-parameters>
+		    <db:output-parameter key="result" type="VARCHAR"></db:output-parameter>
+		</db:output-parameters>
+	</db:stored-procedure>
+----
```
Comment (estebanwasinger) [inline modules/ROOT/pages/db/database-connector-examples.adoc]: How a customer can test this example?
Is there a way to provide the store procedure code?

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 3]
```diff
@@ -490,6 +490,29 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
 
+Database connector supports the use of callable statements with the format { call procedureName(:param1, :param2, ..., :paramN) } where parameters will be matched positionally, that is to say, :paramN will match the Nth parameter in the stored procedure declaration. Suppose you have declared a stored procedure which receives two parameters: myFirstParameter and mySecondParameter, in that order. The Database connector does NOT support named parameters in callable statements, so attempting to match parameters by name ( { call myStoredProcedure(mySecondParameter => :param1, myFirstParameter => :param2) } ) will not work.
+
+It also does not support the use of engine specific embedded functions, for example, you might want to change a date to a specific format before calling a procedure in an Oracle database, however, the following approach will not work:  { call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }. All data transformations should be done using dataweave, before the invocation to the callable statement. For example, for the previous example we would perform the following transformation:
```
Comment (estebanwasinger) [inline modules/ROOT/pages/db/database-connector-examples.adoc]: wrap { call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }
as code snippets `{ call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }`

**Review by estebanwasinger — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/db/database-connector-examples.adoc (hunk 3 above)
Comment (kevintroller) [reply modules/ROOT/pages/db/database-connector-examples.adoc]: Updated.

File: modules/ROOT/pages/db/database-connector-examples.adoc (hunk 1 above)
Comment (kevintroller) [reply modules/ROOT/pages/db/database-connector-examples.adoc]: Updated.

File: modules/ROOT/pages/db/database-connector-examples.adoc (hunk 2 above)
Comment (kevintroller) [reply modules/ROOT/pages/db/database-connector-examples.adoc]: Resolved in commit [59a4c4c](https://github.com/mulesoft/docs-connectors/pull/508/commits/59a4c4c4a212175fa1612625d933c9901ebd1c9a)

**Comment by kevintroller** [comment]
Closing this PR to avoid conflict during merge. New PR https://github.com/mulesoft/docs-connectors/pull/696/files

**Pull request closed without merging**